polyfill/intersection-observer.js

```diff
@@ -368,6 +368,10 @@ IntersectionObserver.prototype._computeTargetAndRootIntersection =
 
   while (!atRoot) {
     var parentRect = null;
+    var parentComputedStyle = window.getComputedStyle(parent);
```

Comment:
Can you go into a bit more detail here? What specifically broke? `getComputedStyle()` should work on the `<html>` element, and if the `<html>` element is set to `display: none`, I think exiting early is still the correct logic.